cfgov/templates/wagtailadmin/pages/create.html

```diff
@@ -18,7 +18,7 @@ <h1 class="icon icon-doc-empty-inverse">{% trans 'New' %} <span>{{ content_type.
         </div>
     </header>
 
-    <form id="page-edit-form" action="{% url 'wagtailadmin_pages:add' content_type.app_label content_type.model parent_page.id %}" method="POST"{% if form.is_multipart %} enctype="multipart/form-data"{% endif %}>
+    <form id="page-edit-form" action="{% url 'wagtailadmin_pages:add' content_type.app_label content_type.model parent_page.id %}" method="POST" novalidate{% if form.is_multipart %} enctype="multipart/form-data"{% endif %}>
```

Comment:
Just a note on this change which adds the `novalidate` attribute to the `<form>` field. This was introduced in Wagtail in [this commit](https://github.com/wagtail/wagtail/commit/6c0f2235a4afcb6b596531c431f84429757d3cbd) which has a pretty good explanation. FEWDs, do we do anything custom with Wagtail editor form submission that this might conflict with?